### ISSUE TITLE
Pillar can't compile just 1 file

### ISSUE THREAD

**Issue by DamienCassou** [pillar-markup/pillar#81]
running `./compile.sh` in books compiles everything.

**Comment by DamienCassou** [comment]
```
Name: Pillar-Cli-DamienCassou.18
Author: DamienCassou
Time: 28 September 2015, 1:13:23.441091 pm
UUID: c0e5a726-da4b-4940-a66a-c2c6464d8d07
Ancestors: Pillar-Cli-DamienCassou.17

Fix #81: we should be able to compile just 1 file from the command line
```

**Comment by DamienCassou** [comment]
This seems fixed. Please execute something like:

```
./compile.sh --to=Beamer Slides/4-Done/Basic-ArraySetOrderedCollection.pillar
```
